### DISCUSSION TITLE
Title Separator should be a text field or a "custom" option should be added.

### DISCUSSION THREAD

**Discussion by el-schneider** [aerni/statamic-advanced-seo#90] [Ideas]
While I'm generally a friend of giving as little control to clients as possible to prevent them from messing up I think the current options are too narrow. There are plenty of viable options for separators not covererd right now. Like en- and em-dashes, `+`, `←`, `→`, `=`, `⋰` or multiples of `/`, `>` or `:`.

Because there are so many good options a `custom` option with a conditional validated text field, or one that replaces the select field would be best. And maybe validation is not even needed, because who am I to prevent my client from using `======` as a separator. If it's what makes them happy ...

**Comment by aerni** [comment]
This is a duplicate of idea #75.

**Comment by el-schneider** [comment]
If I understand correctly, the solution then is to use custom titles all the time then?

**Reply by aerni** [reply]
If you need a separator that doesn't exist as a predefined option, yes. I will only add the option for a custom separator to the existing options in the general site config.

If you want a different separator for a particular entry, you can disable the site on that entry and manually add your preferred separator and the site name as a custom meta title.

**Reply by el-schneider** [reply]
Ah, okay. I think this was a misunderstanding then. Being able to only overwrite seperators on individual entries would surely be overkill. What I meant was what you are saying here. A sitewide option for a custom separator string would be awesome.
